aten/src/ATen/native/cuda/ForeachFunctors.cuh

```diff
@@ -152,49 +152,6 @@ __device__ __forceinline__ void binary_op_scalar(
   }
 }
 
-template <int res_arg_index, typename Op, typename T, typename scalar_t = T>
```

Comment:
I ended up inlining this change due to not being able to specify that scalar is of type opmath_t. I cannot figure out where to put `using opmath_t = at::opmath_type<T>;` before the function signature but after the template specification.